attributes/default.rb

```diff
@@ -1,3 +1,12 @@
 default['osl-php']['packages'] = []
 default['osl-php']['php_packages'] = []
 default['osl-php']['use_ius'] = false
+default['osl-php']['use_opcache'] = false
+
+# defaults from php72u-opcache package on Centos 7
+default['osl-php']['opcache']['enable'] = 1
+default['osl-php']['opcache']['memory_consumption'] = 128
+default['osl-php']['opcache']['interned_strings_buffer'] = 8
+default['osl-php']['opcache']['max_accelerated_files'] = 4000
+default['osl-php']['opcache']['blacklist_filename'] = '/etc/php.d/opcache*.blacklist'
+default['osl-php']['opcache']['huge_code_pages'] = 0
```

Comment:
Let's move all ``default['osl-php']['opcache']`` into its own attribute file called ``opcache.rb``

Reply:
Moved to `attributes/opcache.rb`